docs/admin/pool_operations.md

```diff
@@ -497,6 +531,41 @@ applications. Valid values are 0% to 100%, the default is 0%.
 When setting this property, specifying the percentage symbol is optional:
 `space_rb:2%` and `space_rb:2` both specify two percent of storage capacity.
 
+### Data bdev threshold size (data\_thresh)
+
+This property defines the size threshold over which data is stored on backend block device
+(e.g. NVMe SSDs). Any single value or array value extent of size greater than or
+equal to the data\_thresh value is stored on a block device. It is otherwise stored
+in SCM. 0 is a special value meaning that data is always stored on SCM regardless of
+the size. 1 implies that data is always stored on the block device.
```

Comment:
So setting property to 1 is a special case but value of 2 will be treated as 2byte threshold? if so this sounds like a confusing use of a property.

Reply:
only 0 is special and means all in SCM. 1-infinity is just treated as byte limit.